<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1299</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Plotter" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>411</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="PlanoXY">
       <property name="text">
        <string>PlanoXY</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PlanoXZ">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>85</red>
             <green>255</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Link">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>85</red>
             <green>255</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Link">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>85</red>
             <green>255</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Link">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="text">
        <string>PlanoXZ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PlanoYZ">
       <property name="text">
        <string>PlanoYZ</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Profundidade</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="Profundidade">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="sliderPosition">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Plano XY</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>109</y>
      <width>321</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>R</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLCDNumber" name="GValor">
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLCDNumber" name="RValor">
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLCDNumber" name="BValor">
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="SliderGreen">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>B</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>G</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Cor</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSlider" name="SliderBlue">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="SliderRED">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>230</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="1">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Raio da Esfera</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLCDNumber" name="RaioEsferaValor">
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="SliderRaio">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>R</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>300</y>
      <width>321</width>
      <height>102</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="4" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Z</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLCDNumber" name="DimValorBoxY">
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLCDNumber" name="DimValorBoxX">
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSlider" name="SliderBoxDimZ">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSlider" name="SliderBoxDimY">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLCDNumber" name="DimValorBoxZ">
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="SliderBoxDimX">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Dimensao do Box</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>420</y>
      <width>321</width>
      <height>102</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="4" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Z</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLCDNumber" name="DimValorEllipseY">
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLCDNumber" name="DimValorEllipseX">
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSlider" name="SliderEllipseDimZ">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSlider" name="SliderEllipseDimY">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLCDNumber" name="DimValorEllipseZ">
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="SliderEllipseDimX">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Raio do Ellipsoid</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Cor" name="widgetCor" native="true">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>30</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1299</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSalvar"/>
   <addaction name="actionPutVoxel"/>
   <addaction name="actionCutVoxel"/>
   <addaction name="actionPutBox"/>
   <addaction name="actionPutSphere"/>
   <addaction name="actionPutEllipse"/>
   <addaction name="actionCutBox"/>
   <addaction name="actionCutSphere"/>
   <addaction name="actionCutEllipsoid"/>
  </widget>
  <action name="actionPutBox">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/new/icones/putBoxIcon</normaloff>:/new/icones/putBoxIcon</iconset>
   </property>
   <property name="text">
    <string>PutBox</string>
   </property>
   <property name="toolTip">
    <string>PutBox</string>
   </property>
  </action>
  <action name="actionPutSphere">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/new/icones/putSphereIcon</normaloff>:/new/icones/putSphereIcon</iconset>
   </property>
   <property name="text">
    <string>PutSphere</string>
   </property>
   <property name="toolTip">
    <string>PutSphere</string>
   </property>
  </action>
  <action name="actionPutEllipse">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/new/icones/putEllipsoidIcon</normaloff>:/new/icones/putEllipsoidIcon</iconset>
   </property>
   <property name="text">
    <string>PutEllipse</string>
   </property>
   <property name="toolTip">
    <string>PutEllipse</string>
   </property>
  </action>
  <action name="actionPutVoxel">
   <property name="text">
    <string>P</string>
   </property>
   <property name="toolTip">
    <string>PutVoxel</string>
   </property>
  </action>
  <action name="actionCutBox">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/new/icones/recursos/cutBox.png</normaloff>:/new/icones/recursos/cutBox.png</iconset>
   </property>
   <property name="text">
    <string>CutBox</string>
   </property>
   <property name="toolTip">
    <string>CutBox</string>
   </property>
  </action>
  <action name="actionCutSphere">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/new/icones/recursos/cutSphere.png</normaloff>:/new/icones/recursos/cutSphere.png</iconset>
   </property>
   <property name="text">
    <string>CutSphere</string>
   </property>
   <property name="toolTip">
    <string>CutSphere</string>
   </property>
  </action>
  <action name="actionCutEllipsoid">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/new/icones/recursos/cutEllipsoid.png</normaloff>:/new/icones/recursos/cutEllipsoid.png</iconset>
   </property>
   <property name="text">
    <string>CutEllipsoid</string>
   </property>
   <property name="toolTip">
    <string>CutEllipsoid</string>
   </property>
  </action>
  <action name="actionCutVoxel">
   <property name="text">
    <string>C</string>
   </property>
   <property name="toolTip">
    <string>CutVoxel</string>
   </property>
  </action>
  <action name="actionSalvar">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/new/icones/recursos/salvar.png</normaloff>:/new/icones/recursos/salvar.png</iconset>
   </property>
   <property name="text">
    <string>Salvar</string>
   </property>
   <property name="toolTip">
    <string>Salvar</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Plotter</class>
   <extends>QWidget</extends>
   <header location="global">plotter.h</header>
   <container>1</container>
   <slots>
    <slot>setAmplitude(int)</slot>
    <slot>slot1()</slot>
    <slot>exibeCorSelecionada(int)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>Cor</class>
   <extends>QWidget</extends>
   <header location="global">cor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>SliderRED</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RValor</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>632</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>740</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SliderGreen</sender>
   <signal>valueChanged(int)</signal>
   <receiver>GValor</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>581</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>757</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SliderBlue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>BValor</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>592</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>744</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SliderRaio</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RaioEsferaValor</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>629</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>734</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SliderBoxDimX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DimValorBoxX</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>615</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>744</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SliderBoxDimY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DimValorBoxY</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>617</x>
     <y>375</y>
    </hint>
    <hint type="destinationlabel">
     <x>735</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SliderBoxDimZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DimValorBoxZ</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>666</x>
     <y>412</y>
    </hint>
    <hint type="destinationlabel">
     <x>734</x>
     <y>412</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SliderEllipseDimX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DimValorEllipseX</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>624</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel">
     <x>759</x>
     <y>477</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SliderEllipseDimY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DimValorEllipseY</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>582</x>
     <y>505</y>
    </hint>
    <hint type="destinationlabel">
     <x>734</x>
     <y>502</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SliderEllipseDimZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DimValorEllipseZ</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>645</x>
     <y>529</y>
    </hint>
    <hint type="destinationlabel">
     <x>746</x>
     <y>532</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
